### PR TITLE
Initial commit of xlf files for localization

### PR DESCRIPTION
**Pull request by nguerrera**
Bootstrapped using https://github.com/nguerrera/xliff-converter to meet fast approaching localization hand-off deadline.

This change in itself does not change the product, it just unblocks the loc team that will handle sending PRs to these xlf files with appropriate translations. Follow up work is still required to actually build and distribute satellite assemblies from the translated data. 

@piotrpMSFT

### CODE REVIEW

**Comment by nguerrera** [comment]
Test CentOS7.1 x64 Debug Build please

**Comment by nguerrera** [comment]
@livarcocc Any idea why CentOS failed twice. There are no code changes here.

**Comment by nguerrera** [comment]
@vsccarl @piotrpMSFT I see a bunch more strings coming in via PRs. Are those the last of them? We'll need to rerun my tool after those are merged. This needs to be done ASAP.

@nataliatgit FYI.

**Comment by vsccarl** [comment]
@nguerrera  The final changes are all in now.

File: src/dotnet/commands/dotnet-test/xlf/LocalizableStrings.it.xlf [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                                        Example: --testAdapterPath:&lt;pathToCustomAdapters&gt;</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="CmdLoggerOption">
+        <source>LoggerUri/FriendlyName</source>
+        <target state="new">LoggerUri/FriendlyName</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="CmdLoggerDescription">
+        <source>@"Specify a logger for test results.
```
Comment (nguerrera) [inline src/dotnet/commands/dotnet-test/xlf/LocalizableStrings.it.xlf]: There's a bug in my tool here for @ prefixed strings. I'll fix it.

Comment (nguerrera) [inline src/dotnet/commands/dotnet-test/xlf/LocalizableStrings.it.xlf]: Fixed

**Comment by piotrpMSFT** [comment]
@MattGertz can you pre-approve this change so we can merge as soon as it is green?

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Debian8.2 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OpenSUSE13.2 x64 Debug Build please.

**Comment by MattGertz** [comment]
Approved pending tests passing.

**Comment by nguerrera** [comment]
@piotrpMSFT Thanks for merging. Note that we need to run the tool again once all the new strings in process in PRs are merged.

**Pull request merged**